src/GraphQL.Client/Websocket/GraphQLHttpWebSocket.cs

```diff
@@ -468,9 +456,38 @@ private async Task ConnectAsync(CancellationToken token)
                 Debug.WriteLine($"new incoming message stream {_incomingMessages.GetHashCode()} created");
 
                 _incomingMessagesConnection = new CompositeDisposable(maintenanceSubscription, connection);
+                
+                var initRequest = new GraphQLWebSocketRequest
+                {
+                    Type = GraphQLWebSocketMessageType.GQL_CONNECTION_INIT,
+                    Payload = new GraphQLRequest()
```

Comment:
new GraphQLRequest() ? It seems like there could be any type of payload.

Reply:
yeah, that should be replaced by the payload from #294... I'll fix this as soon as the former gets merged...

Reply:
Just didn't want to mix them up

Reply:
ok